tests/test_catalyst/test_openinference_langchain.py

```
@@ -0,0 +1,562 @@
+        chain = prompt | model | parser
+        output = chain.invoke({})
+        print(f"Raw LLM Response: {output}")
+    except Exception as e:
+        logger.warning(f"Errors during chain execution: {e}")
+        try:
+            chain = prompt | model
+            output = chain.invoke({})
+            print(f"Raw LLM Response: here::::{output}")
+            try:
+                wrapped_resp = json.dumps({"impression": output})
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)